File: calculadora.rb
while true
    puts "========================================="
    puts "========== CALCULADORA EN RUBY =========="
    puts "========================================="

    #Ingreso de datos
    puts "Ingresa el primer número: " 
    a = gets().to_i
    puts "Ingresa el segundo número: "
    b = gets().to_i

    #Proceso

    #Suma de dos números
    def suma(a, b)
        c = a + b
        puts "El resultado es: ",c
    end

    #Resta de dos números
    def resta(a, b)
        c = a - b
        puts "El resultado es: ",c
    end

    #Multiplicación de dos números
    def multi(a, b)
        c = a * b
        puts "El resultado es: ",c
    end

    #División de dos números
    def divi(a, b)
        c = a / b
        puts "El resultado es: ",c
    end

    puts "¿Que desea hacer?"
    puts "1- Sumar"
    puts "2- Restar"
    puts "3- Multiplicar"
    puts "4- Dividir"

    #Asignación
    rta = gets().to_i
    if rta == 1
        suma(a, b)
    elsif rta == 2
        resta(a, b)
    elsif rta == 3
        multi(a, b)
    elsif rta == 4
        divi(a, b)
    else
        puts "Opción invalida"
    end

end

File: calculadora.py
while True:
    print("=========================================") 
    print("========= CALCULADORA EN PYTHON =========")
    print("=========================================")

    #Ingreso de datos
    a = int(input("Ingresa el primer número: "))
    b = int(input("Ingresa el segundo número: "))

    #Proceso

    #Suma de dos números
    def suma(a, b):
        c = a + b
        print("El resultado es: ",c)


    #Resta de dos números
    def resta(a, b):
        c = a - b
        print("El resultado es: ",c)


    #Multiplicación de dos números
    def multi(a, b):
        c = a * b
        print("El resultado es: ",c)


    #División de dos números
    def divi(a, b):
        c = a / b
        print("El resultado es: ",c)


    print("¿Que desea hacer?")
    print("1- Sumar")
    print("2- Restar")
    print("3- Multiplicar")
    print("4- Dividir")

    #Asignación
    rta = int(input(":"))
    if rta == 1:
        suma(a, b)
    elif rta == 2:
        resta(a, b)
    elif rta == 3:
        multi(a, b)
    elif rta == 4:
        divi(a, b)
    else:
        print("Opción invalida")